src/main/webapp/dataset.xhtml

```diff
@@ -43,7 +43,7 @@
                 <meta property="og:title" content="#{DatasetPage.title}" />
                 <meta property="og:type" content="article" />
                 <meta property="og:url" content="#{DatasetPage.dataverseSiteUrl}/dataset.xhtml?persistentId=#{dataset.globalId}" />
-                <meta property="og:image" content="#{DatasetPage.dataverseSiteUrl.concat(resource['images/favicondataverse.png'])}" />
+                <meta property="og:image" content="#{DatasetPage.dataset.getDatasetThumbnail() == null ? DatasetPage.dataverseSiteUrl.concat(resource['images/favicondataverse.png']): DatasetPage.dataverseSiteUrl.concat('/api/datasets/:persistentId/thumbnail?persistentId=').concat(DatasetPage.dataset.getGlobalId().asString())}" />
```

Comment:
@qqmyers -- one question I have about this `og:image` file path is if it requires the `dataverseSiteUrl` in your installation? In my initial local and AWS tests, it seems that a relative path with just `content="#{resource['images/...']}"` to the default image file works with no problem.

Reply:
@mheppler - I'm not sure how you're testing, but it may be that relative URLs are OK for some uses. I think we went with a full URL based on https://stackoverflow.com/a/9858694 and looking at https://developers.facebook.com/tools/debug/og/object/ which gives an error such as 
![image](https://user-images.githubusercontent.com/6731983/54373070-00227980-4653-11e9-8fb7-d44a3429306e.png)
with a relative URL. 

That said, Facebook also complains about the article:author field not being a profile (minimally a URL for the person), and doesn't like images less than 200x200... other things that could/should be fixed but that don't appear to cause trouble in slack for example...

Reply:
Ah, that might explain the lack of complaints on my end. I haven't checked my local development on the Facebook yet. (That's on today's agenda.) The dataset thumbnail image size was also something I was going to experiment today, so I am glad to hear you've already gone down that route.